build.gradle

```diff
@@ -22,3 +23,8 @@ test {
         events "passed", "skipped", "failed"
     }
 }
+
+javafx {
+    version = '12'
+    modules = [ 'javafx.controls', 'javafx.fxml', 'javafx.base', 'javafx.graphics', 'javafx.media' ]
+}
```

Comment:
You have not completely configured gradle properly... it should get to the point where you only have to run `./gradlew run`, and that will download all the dependencies automatically.